### PR TITLE
fix/mastertrust_page_hidden

### PR DESCRIPTION
**Pull request by hemendra-sakpal-algobulls**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -74,6 +74,7 @@ nav:
           - 'Trade Book': 'member/Books/tradebook.md'
           - 'Fund Book': 'member/Books/fundbook.md'
       - 'Broking Details': 'member/broking-details.md'
+      - 'Fixed Income': 'member/fixed-income.md'
```
Comment (pushpakdagade) [inline mkdocs.yml:77]: why this change?

Comment (hemendra-sakpal-algobulls) [reply mkdocs.yml:77]: This is for the Fixed Income page though the page is not pushed yet so it won't show on the site

Comment (hemendra-sakpal-algobulls) [reply mkdocs.yml:77]: Since we want to add that page to the help site anyways, I thought of adding it

Comment (hemendra-sakpal-algobulls) [reply mkdocs.yml:77]: Ok

**Review by pushpakdagade — APPROVED** [review]
